src/house.cpp

```diff
@@ -162,7 +162,7 @@ void House::setAccessList(uint32_t listId, std::string_view textlist)
 	} else if (listId == SUBOWNER_LIST) {
 		subOwnerList.parseList(textlist);
 	} else {
-		Door* door = getDoorByNumber(listId);
+		auto door = getDoorByNumber(listId);
```

Comment:
Could this be `const auto`
```suggestion
		cont auto door = getDoorByNumber(listId);
```